.github/workflows/ci.yml

```diff
@@ -62,8 +62,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.11]
-        os: [ubuntu-latest, macos-latest]
+        python-version: ["3.10", 3.11]
+        os: [ubuntu-latest]
```

Comment:
bit of a smuggle, but the macos builds are causing lots of issues, and I was reminded that 3.10 ought really to be the target for ray